app/src/main/java/com/example/userinfo/DisplayUserInfo.kt

```diff
@@ -0,0 +1,20 @@
+package com.example.userinfo
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.widget.TextView
+
+class DisplayUserInfo : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_display_user_info)
+        val userName = intent.getStringExtra(UserName)
+        val email= intent.getStringExtra(Email)
+        val address = intent.getStringExtra(Address)
+        val phoneNumber = intent.getStringExtra(PhoneNumber)
+        val pinCode=intent.getStringExtra(PinCode)
+        val message="Hi "+userName+", How are you? Are you staying at "+ address +"-"+pinCode+"code. I am not able to contact you on"+phoneNumber+". Can I " +
```

Comment:
We can use string templates : https://stackoverflow.com/questions/3656371/is-it-possible-to-have-placeholders-in-strings-xml-for-runtime-values